user-stories.md

```diff
@@ -0,0 +1,9 @@
+As a reminder, user stories typically take the form of, "As X, I want Y, so that Z".
+
+
+As someone who rides the bus I want to have an app where I can shop so that I can 
```

Comment:
Give a bit more for your user stories! Something like "Elliot has a long commute and takes the bus to and from work. With the holidays coming up, they need to buy quirky gifts for their friends. With bus-mall, Elliot is able to search for gifts while on their commute"